$NetBSD: patch-Makefile.libretro,v 1.1 2018/08/08 22:14:20 nia Exp $

Picodrive is written in C. Let's link with a C compiler instead of
a C++ compiler.

--- Makefile.libretro.orig	2018-07-22 20:01:05.000000000 +0000
+++ Makefile.libretro
@@ -656,7 +656,7 @@ else
 	LD = link.exe
 endif
 else
-	LD = $(CXX)
+	LD = $(CC)
 endif
 
 include Makefile
